core/views.py

```diff
@@ -708,3 +733,590 @@ def spoof_user_search(request):
         "partials/spoof_user_options_for_select.html", {"users": users}
     )
     return HttpResponse(html)
+
+
+# TODO add permission check
```

Comment:
The is a really important TODO. Maybe make an issue and add it to the MVP milestone?